roles/edpm_iscsid/tasks/run.yml

```diff
@@ -52,7 +56,6 @@
     state: restarted
 
 - name: Remove iscsid container restart sentinel file
-  become: true
```

Comment:
this is created with `become: true` in the handler. why drop it here?

Reply:
No reason, I must have removed it accidentally.